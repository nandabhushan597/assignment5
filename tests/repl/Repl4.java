"hi"
"there"
  :heap
  :heap 2
  :heap 2
  :heap 5
  :heap 0
  :heap garbage
  :quit
