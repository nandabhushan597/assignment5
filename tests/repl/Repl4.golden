Exit code: ExitSuccess
Stdout:
Welcome to the Java interpreter.
Type `:?` at the prompt for the list of commands.
Java> @2 => "hi"
Java> @4 => "there"
Java> @1 => char[2] { 'h', 'i' }
@2 => String { value = @1 => char[2] { 'h', 'i' } }
@3 => char[5] { 't', 'h', 'e', 'r', 'e' }
@4 => String { value = @3 => char[5] { 't', 'h', 'e', 'r', 'e' } }
Java> @2 => String { value = @1 => char[2] { 'h', 'i' } }
Java> @2 => String { value = @1 => char[2] { 'h', 'i' } }
Java> Heap pointer too high: 5
Java> Heap pointer too low: 0
Java> Unknown heap element:  garbage
Java> Good-bye.

Stderr:
